Signal/src/Models/TSMessageAdapaters/TSAnimatedAdapter.m

```diff
@@ -21,6 +21,7 @@ @interface TSAnimatedAdapter ()
 @property (nonatomic) TSAttachmentStream *attachment;
 @property (nonatomic, nullable) AttachmentUploadView *attachmentUploadView;
 @property (nonatomic) BOOL incoming;
+@property (nonatomic, nullable, weak) id lastCell;
```

Comment:
nit: maybe "owningCell" or "presentingCell" or something? "last" is sort of ambiguous outside of the context of this PR.